Apps/WebClient/src/ClientApp/src/components/site/HeaderComponent.vue

```diff
@@ -9,21 +9,20 @@ import RatingComponent from "@/components/site/RatingComponent.vue";
 import { container } from "@/ioc/container";
 import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
 import { ILogger } from "@/services/interfaces";
-import { useAppStore } from "@/stores/app";
 import { useAuthStore } from "@/stores/auth";
 import { useConfigStore } from "@/stores/config";
-import { useNavigationStore } from "@/stores/navigation";
+import { useLayoutStore } from "@/stores/layout";
 import { useNotificationStore } from "@/stores/notification";
 import { useUserStore } from "@/stores/user";
 
 const headerScrollThreshold = 100;
 
-const appStore = useAppStore();
+const layoutStore = useLayoutStore();
 const configStore = useConfigStore();
 const userStore = useUserStore();
 const notificationStore = useNotificationStore();
 const authStore = useAuthStore();
-const navigationStore = useNavigationStore();
+const navigationStore = useLayoutStore();
```

Comment:
The `const navigationStore` line won't be needed if other references to `navigationStore` in the file are replaced with `layoutStore`.

Reply:
Fixed